pkg/util/debug_logging_test.go

```diff
@@ -132,10 +132,18 @@ func TestDebugLoggerThreadSafety(t *testing.T) {
 	log := MakeDebugLoggerWithReducedRedudancy(buf, &logrus.JSONFormatter{}, 0, time.Minute*30)
 
 	// spam the logger concurrently across several goroutines to ensure no dataraces
+	wg := &sync.WaitGroup{}
 	for i := 0; i < 100; i++ {
-		go func() { log.Debug("unique") }()
+		wg.Add(1)
```

Comment:
nit: just `Add` once and put `100` in a constant

Reply:
changed: 73ad9ebd8bd66adb1e9266510e8f97a549f5e159